### PR TITLE
Maint mergefix module/halos

### PR DESCRIPTION
**Pull request by itrharrison**
## Checklist
- [ ] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/main/CONTRIBUTING.rst)
- [ ] Write unit tests
- [ ] Write documentation strings
- [ ] Assign someone from your working team to review this pull request
- [ ] Assign someone from the infrastructure team to review this pull request

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -42,14 +42,23 @@ description =
     numpy117: with numpy 1.17.*
     numpy118: with numpy 1.18.*
     numpy119: with numpy 1.19.*
+    numpy120: with numpy 1.20.*
+    numpy121: with numpy 1.21.*
+    numpy122: with numpy 1.22.*
+    numpy123: with numpy 1.23.*
     scipy12: with scipy 1.2.*
```
Comment (jucordero) [inline tox.ini:49]: Add numpy 1.24, scipy 1.10 and astropy 5.2 here?

**Review by jucordero — CHANGES_REQUESTED** [review]
I believe latest versions of some packages need to be updated.
I don't know hot to add code suggestions but added a small comment on the tox.ini file

File: tox.ini (hunk 1 above)
Comment (rrjbca) [reply tox.ini:49]: Yeah I don't think this branch is up to date with main

Comment (rrjbca) [reply tox.ini:49]: I also think the merge conflict resolution went awry, some the the commits done afterwards make changes that were already on main (codestyle and passenv)

**Comment by rrjbca** [comment]
Closed in favour of #596

**Pull request closed without merging**